theano/tests/test_tutorial.py

```diff
@@ -139,16 +139,13 @@ def make_node(x, y):
 
         from theano import gof
         class BinaryDoubleOp(gof.Op):
+        
+            __props__ = ("name", "fn")
+            
```

Comment:
I think it would be better to update the tutorials instead. The files are in `doc/extending/*.txt`.
This applies to this whole file.